readthedocs/builds/version_slug.py

```diff
@@ -26,6 +26,7 @@
 
 from django.db import models
 from django.utils.encoding import force_text
+from slugify import slugify
```

Comment:
I wonder if we should have a different import for this:

```suggestion
from slugify import slugify as unicode_slugify
```